lib/utils.pm

```diff
@@ -1599,6 +1613,9 @@ test user as well as root.
 sub ensure_serialdev_permissions {
     my ($self) = @_;
     return if get_var('ROOTONLY');
+
+    ensure_testuser_present;
```

Comment:
Is it really needed to be here? Cannot be in `prepare_serial_console()` which so far did all needed preparations?

Reply:
Yes it needs to be here, because `ensure_serialdev_permissions` needs to be called before `prepare_serial_console`, otherwise one gets permission denied errors, and because `ensure_serialdev_permissions` needs the user to be present, otherwise the `chown` command fails.